src/app/Hooks/HooksPage.tsx

```diff
@@ -0,0 +1,93 @@
+import * as React from 'react';
+import {
+  Card,
+  PageSection,
+  CardBody,
+  EmptyState,
+  EmptyStateIcon,
+  EmptyStateBody,
+  Title,
+  Button,
+} from '@patternfly/react-core';
+import spacing from '@patternfly/react-styles/css/utilities/Spacing/spacing';
+import { PlusCircleIcon } from '@patternfly/react-icons';
+
+import {
+  useHasSufficientProvidersQuery,
+  useHooksQuery,
+  useClusterProvidersQuery,
+} from '@app/queries';
+
+import HooksTable from './components/HooksTable';
+import { ResolvedQueries } from '@app/common/components/ResolvedQuery';
+import { IHook } from '@app/queries/types';
+import AddEditHookModal from './components/AddEditHookModal';
+
+const HooksPage: React.FunctionComponent = () => {
+  const sufficientProvidersQuery = useHasSufficientProvidersQuery();
+  const clusterProvidersQuery = useClusterProvidersQuery();
```

Comment:
I don't think we need these providers queries in here necessarily, right?

Reply:
You're right, that's left over ;)